cli/src/vote.rs

```diff
@@ -276,6 +317,35 @@ pub fn parse_vote_get_account_command(
     })
 }
 
+pub fn parse_vote_account_withdraw_command(
+    matches: &ArgMatches<'_>,
+    default_signer_path: &str,
+    wallet_manager: Option<&Arc<RemoteWalletManager>>,
+) -> Result<CliCommandInfo, CliError> {
+    let vote_account_pubkey = pubkey_of(matches, "vote_account_pubkey").unwrap();
+    let withdrawer_account_pubkey = pubkey_of(matches, "withdrawer_account_pubkey").unwrap();
+    let lamports: u64 = matches.value_of("lamports").unwrap().parse().unwrap();
+    let to_account_pubkey = pubkey_of(matches, "to_account_pubkey").unwrap();
+
```

Comment:
I don't have an account to easily test this -- feedback would be super useful from someone who can actually try this out...

Reply:
✔️ Integration test added!